salt/transport/tcp.py

```diff
@@ -42,6 +41,13 @@
 import tornado.tcpclient
 import tornado.netutil
 
+# pylint: disable=import-error,no-name-in-module
+if six.PY2:
```

Comment:
Since I am not familiar with `six.moves`, I will keep it as is unless you request otherwise.